scipy/odr/__odrpack.c

```diff
@@ -9,6 +9,8 @@
  *
  */
 
+#define PY_SSIZE_T_CLEAN
+#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
```

Comment:
This shouldn't be defined here, but in `setup.py` so we can change it centrally. See for example https://github.com/scipy/scipy/blob/master/scipy/signal/setup.py#L21